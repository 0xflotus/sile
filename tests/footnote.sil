\begin[papersize=a5,class=book]{document}
\script[src=packages/rules]
\footnote:separator{\em{Separator}\skip[height=5mm]}
\footnote:options[interInsertionSkip=3mm]
Paragraph one.\footnote{Footnote one}

Paragraph two.\footnote{Footnote two. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.}

Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.

Paragraph three.\footnote{Footnote three. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.}

Paragraph four.\footnote{Footnote four}\par
Paragraph five.\footnote{Footnote five. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.}

Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.

Paragraph six.\footnote{Footnote six. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.}
\end{document}
