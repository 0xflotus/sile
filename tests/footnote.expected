<tests/footnote.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	54.82079
My 	76.62033
Set font 	Gentium;10;200;regular;normal;;LTR
T	48 65 82 65 71 82 65 80 72	(Paragraph)
Mx 	100.10210
T	79 78 69 14	(one.)
Mx 	117.55816
My 	73.44162
Set font 	Gentium;6.81152;200;regular;normal;;LTR
T	17	(1)
Mx 	54.82079
My 	89.62033
Set font 	Gentium;10;200;regular;normal;;LTR
T	48 65 82 65 71 82 65 80 72	(Paragraph)
Mx 	100.10206
T	84 87 79 14	(two.)
Mx 	117.92433
My 	86.44162
Set font 	Gentium;6.81152;200;regular;normal;;LTR
T	18	(2)
Mx 	54.82079
My 	102.62033
Set font 	Gentium;10;200;regular;normal;;LTR
T	44 79 82 69 77	(Lorem)
Mx 	84.41564
T	73 80 83 85 77	(ipsum)
Mx 	112.80932
T	68 79 76 79 82	(dolor)
Mx 	137.93151
T	83 73 84	(sit)
Mx 	151.13964
T	65 77 69 84 12	(amet,)
Mx 	177.30676
T	67 79 78 83 69 67 84 69 84 85 82	(consectetur)
Mx 	229.03051
T	65 68 73 80 73 83 73 67 73 78 71	(adipisicing)
Mx 	276.77966
T	69 76 73 84 12	(elit,)
Mx 	295.74463
T	83 69 68	(sed)
Mx 	312.61975
T	68 79	(do)
Mx 	326.04273
T	69 73 85 83 77 79 68	(eiusmod)
Mx 	34.82079
My 	115.62033
T	84 69 77 80 79 82	(tempor)
Mx 	67.55933
T	73 78 67 73 68 73 68 85 78 84	(incididunt)
Mx 	112.59280
T	85 84	(ut)
Mx 	123.69073
T	76 65 66 79 82 69	(labore)
Mx 	152.03474
T	69 84	(et)
Mx 	162.45395
T	68 79 76 79 82 69	(dolore)
Mx 	191.35949
T	77 65 71 78 65	(magna)
Mx 	221.32460
T	65 76 73 81 85 65 14	(aliqua.)
Mx 	251.02116
T	53 84	(Ut)
Mx 	263.35931
T	69 78 73 77	(enim)
Mx 	286.59591
T	65 68	(ad)
Mx 	298.74363
T	77 73 78 73 77	(minim)
Mx 	328.10328
T	86 69 78 73 65 77 12	(veniam,)
Mx 	34.82079
My 	128.62033
T	81 85 73 83	(quis)
Mx 	54.34860
T	78 79 83 84 82 85 68	(nostrud)
Mx 	89.16450
T	69 88 69 82 67 73 84 65 84 73 79 78	(exercitation)
Mx 	141.70989
T	85 76 76 65 77 67 79	(ullamco)
Mx 	176.95547
T	76 65 66 79 82 73 83	(laboris)
Mx 	207.40125
T	78 73 83 73	(nisi)
Mx 	224.70738
T	85 84	(ut)
Mx 	235.95394
T	65 76 73 81 85 73 80	(aliquip)
Mx 	266.92707
T	69 88	(ex)
Mx 	279.08183
T	69 65	(ea)
Mx 	290.79714
T	67 79 77 77 79 68 79	(commodo)
Mx 	334.02613
T	67 79 78	(con)
Mx 	348.94312
T	83 69	(se)
Mx 	357.42456
T	13	(-)
Mx 	34.82079
My 	141.62033
T	81 85 65 84 14	(quat.)
Mx 	57.70623
T	36 85 73 83	(Duis)
Mx 	77.61803
T	65 85 84 69	(aute)
Mx 	97.68120
T	73 82 85 82 69	(irure)
Mx 	120.34203
T	68 79 76 79 82	(dolor)
Mx 	144.38470
T	73 78	(in)
Mx 	154.72619
T	82 69 80 82 69 72 69 78 68 69 82 73 84	(reprehenderit)
Mx 	214.88214
T	73 78	(in)
Mx 	225.22363
T	86 79 76 85 80 84 65 84 69	(voluptate)
Mx 	266.69794
T	86 69 76 73 84	(velit)
Mx 	287.22498
T	69 83 83 69	(esse)
Mx 	306.30182
T	67 73 76 76 85 77	(cillum)
Mx 	334.24585
T	68 79 76 79 82 69	(dolore)
Mx 	34.82079
My 	154.62033
T	69 85	(eu)
Mx 	47.87413
T	70 85 71 73 65 84	(fugiat)
Mx 	75.03881
T	78 85 76 76 65	(nulla)
Mx 	99.00036
T	80 65 82 73 65 84 85 82 14	(pariatur.)
Mx 	138.27441
T	37 88 67 69 80 84 69 85 82	(Excepteur)
Mx 	183.00744
T	83 73 78 84	(sint)
Mx 	201.67602
T	79 67 67 65 69 67 65 84	(occaecat)
Mx 	240.19324
T	67 85 80 73 68 65 84 65 84	(cupidatat)
Mx 	282.27002
T	78 79 78	(non)
Mx 	301.47083
T	80 82 79 73 68 69 78 84 12	(proident,)
Mx 	342.67359
T	83 85 78 84	(sunt)
Mx 	34.82079
My 	167.62033
T	73 78	(in)
Mx 	45.68852
T	67 85 76 80 65	(culpa)
Mx 	70.59434
T	81 85 73	(qui)
Mx 	86.39371
T	79 70 70 73 67 73 65	(officia)
Mx 	114.66379
T	68 69 83 69 82 85 78 84	(deserunt)
Mx 	153.82254
T	77 79 76 76 73 84	(mollit)
Mx 	181.09652
T	65 78 73 77	(anim)
Mx 	204.58633
T	73 68	(id)
Mx 	215.13668
T	69 83 84	(est)
Mx 	229.70070
T	76 65 66 79 82 85 77 14	(laborum.)
Mx 	269.32820
T	44 79 82 69 77	(Lorem)
Mx 	298.36976
T	73 80 83 85 77	(ipsum)
Mx 	326.21015
T	68 79 76 79 82	(dolor)
Mx 	350.77906
T	83 73 84	(sit)
Mx 	34.82079
My 	180.62033
T	65 77 69 84 12	(amet,)
Mx 	60.56408
T	67 79 78 83 69 67 84 69 84 85 82	(consectetur)
Mx 	111.86400
T	65 68 73 80 73 83 73 67 73 78 71	(adipisicing)
Mx 	159.18932
T	69 76 73 84 12	(elit,)
Mx 	177.73046
T	83 69 68	(sed)
Mx 	194.18176
T	68 79	(do)
Mx 	207.18091
T	69 73 85 83 77 79 68	(eiusmod)
Mx 	244.70154
T	84 69 77 80 79 82	(tempor)
Mx 	277.85205
T	73 78 67 73 68 73 68 85 78 84	(incididunt)
Mx 	323.29749
T	85 84	(ut)
Mx 	334.80738
T	76 65 66 79 82 69	(labore)
Mx 	34.82079
My 	193.62033
T	69 84	(et)
Mx 	44.99301
T	68 79 76 79 82 69	(dolore)
Mx 	73.65156
T	77 65 71 78 65	(magna)
Mx 	103.36968
T	65 76 73 81 85 65 14	(aliqua.)
Mx 	132.81925
T	53 84	(Ut)
Mx 	144.91041
T	69 78 73 77	(enim)
Mx 	167.90002
T	65 68	(ad)
Mx 	179.80076
T	77 73 78 73 77	(minim)
Mx 	208.91341
T	86 69 78 73 65 77 12	(veniam,)
Mx 	243.71454
T	81 85 73 83	(quis)
Mx 	262.84672
T	78 79 83 84 82 85 68	(nostrud)
Mx 	297.26699
T	69 88 69 82 67 73 84 65 84 73 79 78	(exercitation)
Mx 	349.41675
T	85 76	(ul)
Mx 	357.42456
T	13	(-)
Mx 	34.82079
My 	206.62033
T	76 65 77 67 79	(lamco)
Mx 	61.94416
T	76 65 66 79 82 73 83	(laboris)
Mx 	92.27553
T	78 73 83 73	(nisi)
Mx 	109.46726
T	85 84	(ut)
Mx 	120.59941
T	65 76 73 81 85 73 80	(aliquip)
Mx 	151.45813
T	69 88	(ex)
Mx 	163.49849
T	69 65	(ea)
Mx 	175.09940
T	67 79 77 77 79 68 79	(commodo)
Mx 	218.21398
T	67 79 78 83 69 81 85 65 84 14	(consequat.)
Mx 	264.77582
T	36 85 73 83	(Duis)
Mx 	284.96559
T	65 85 84 69	(aute)
Mx 	305.30674
T	73 82 85 82 69	(irure)
Mx 	328.24553
T	68 79 76 79 82	(dolor)
Mx 	352.56617
T	73 78	(in)
Mx 	34.82079
My 	219.62033
T	82 69 80 82 69 72 69 78 68 69 82 73 84	(reprehenderit)
Mx 	95.43747
T	73 78	(in)
Mx 	106.23969
T	86 79 76 85 80 84 65 84 69	(voluptate)
Mx 	148.17473
T	86 69 76 73 84	(velit)
Mx 	169.16250
T	69 83 83 69	(esse)
Mx 	188.70008
T	67 73 76 76 85 77	(cillum)
Mx 	217.10484
T	68 79 76 79 82 69	(dolore)
Mx 	246.22738
T	69 85	(eu)
Mx 	258.71906
T	70 85 71 73 65 84	(fugiat)
Mx 	285.32207
T	78 85 76 76 65	(nulla)
Mx 	308.72195
T	80 65 82 73 65 84 85 82 14	(pariatur.)
Mx 	347.43433
T	37 88	(Ex)
Mx 	357.42456
T	13	(-)
Mx 	34.82079
My 	232.62033
T	67 69 80	(cep)
Mx 	49.10790
T	84 69 85 82	(teur)
Mx 	68.83280
T	83 73 78 84	(sint)
Mx 	86.77059
T	79 67 67 65 69 67 65 84	(occaecat)
Mx 	124.55701
T	67 85 80 73 68 65 84 65 84	(cupidatat)
Mx 	165.90300
T	78 79 78	(non)
Mx 	184.37302
T	80 82 79 73 68 69 78 84 12	(proident,)
Mx 	224.84499
T	83 85 78 84	(sunt)
Mx 	245.37067
T	73 78	(in)
Mx 	256.00377
T	67 85 76 80 65	(culpa)
Mx 	280.67496
T	81 85 73	(qui)
Mx 	296.23970
T	79 70 70 73 67 73 65	(officia)
Mx 	324.27515
T	68 69 83 69 82 85 78 84	(deserunt)
Mx 	34.82079
My 	245.62033
T	77 79 76 76 73 84	(mollit)
Mx 	62.12061
T	65 78 73 77	(anim)
Mx 	85.63625
T	73 68	(id)
Mx 	96.21244
T	69 83 84	(est)
Mx 	110.80230
T	76 65 66 79 82 85 77 14	(laborum.)
Mx 	54.82079
My 	258.62033
T	48 65 82 65 71 82 65 80 72	(Paragraph)
Mx 	100.10133
T	84 72 82 69 69 14	(three.)
Mx 	124.54957
My 	255.44162
Set font 	Gentium;6.81152;200;regular;normal;;LTR
T	19	(3)
Mx 	54.82079
My 	272.25476
Set font 	Gentium;10;200;italic;normal;;LTR
T	51 69 80 65 82 65 84 79 82	(Separator)
Mx 	34.82079
My 	294.49439
Set font 	Gentium;9;200;regular;normal;;LTR
T	17 14	(1.)
Mx 	59.10497
T	38 79 79 84 78 79 84 69	(Footnote)
Mx 	94.70899
T	79 78 69	(one)
Mx 	34.82079
My 	317.18485
T	18 14	(2.)
Mx 	59.10497
T	38 79 79 84 78 79 84 69	(Footnote)
Mx 	95.67743
T	84 87 79 14	(two.)
Mx 	115.08926
T	44 79 82 69 77	(Lorem)
Mx 	142.22227
T	73 80 83 85 77	(ipsum)
Mx 	168.27423
T	68 79 76 79 82	(dolor)
Mx 	191.38185
T	83 73 84	(sit)
Mx 	203.76682
T	65 77 69 84 12	(amet,)
Mx 	227.81487
T	67 79 78 83 69 67 84 69 84 85 82	(consectetur)
Mx 	274.86390
T	65 68 73 80 73 83 73 67 73 78 71	(adipisicing)
Mx 	318.33578
T	69 76 73 84 12	(elit,)
Mx 	335.90190
T	83 69 68	(sed)
Mx 	351.58716
T	68 79	(do)
Mx 	34.82079
My 	330.18485
T	69 73 85 83 77 79 68	(eiusmod)
Mx 	68.15701
T	84 69 77 80 79 82	(tempor)
Mx 	97.56012
T	73 78 67 73 68 73 68 85 78 84	(incididunt)
Mx 	138.02867
T	85 84	(ut)
Mx 	147.95522
T	76 65 66 79 82 69	(labore)
Mx 	173.40326
T	69 84	(et)
Mx 	182.71897
T	68 79 76 79 82 69	(dolore)
Mx 	208.67238
T	77 65 71 78 65	(magna)
Mx 	235.57940
T	65 76 73 81 85 65 14	(aliqua.)
Mx 	262.24472
T	53 84	(Ut)
Mx 	273.28748
T	69 78 73 77	(enim)
Mx 	294.13884
T	65 68	(ad)
Mx 	305.01022
T	77 73 78 73 77	(minim)
Mx 	331.37232
T	86 69 78 73 65 77 12	(veniam,)
Mx 	34.82079
My 	343.18485
T	81 85 73 83	(quis)
Mx 	52.28427
T	78 79 83 84 82 85 68	(nostrud)
Mx 	83.50703
T	69 88 69 82 67 73 84 65 84 73 79 78	(exercitation)
Mx 	130.68633
T	85 76 76 65 77 67 79	(ullamco)
Mx 	162.29581
T	76 65 66 79 82 73 83	(laboris)
Mx 	189.58546
T	78 73 83 73	(nisi)
Mx 	205.04943
T	85 84	(ut)
Mx 	215.05979
T	65 76 73 81 85 73 80	(aliquip)
Mx 	242.82405
T	69 88	(ex)
Mx 	253.65179
T	69 65	(ea)
Mx 	264.08403
T	67 79 77 77 79 68 79	(commodo)
Mx 	302.87856
T	67 79 78 83 69 81 85 65 84 14	(consequat.)
Mx 	344.77564
T	36 85 73 83	(Duis)
Mx 	34.82079
My 	356.18485
T	65 85 84 69	(aute)
Mx 	54.11716
T	73 82 85 82 69	(irure)
Mx 	75.75142
T	68 79 76 79 82	(dolor)
Mx 	98.62933
T	73 78	(in)
Mx 	109.17619
T	82 69 80 82 69 72 69 78 68 69 82 73 84	(reprehenderit)
Mx 	164.55605
T	73 78	(in)
Mx 	175.10291
T	86 79 76 85 80 84 65 84 69	(voluptate)
Mx 	213.66930
T	86 69 76 73 84	(velit)
Mx 	233.38315
T	69 83 83 69	(esse)
Mx 	251.79182
T	67 73 76 76 85 77	(cillum)
Mx 	278.18096
T	68 79 76 79 82 69	(dolore)
Mx 	305.21610
T	69 85	(eu)
Mx 	317.28347
T	70 85 71 73 65 84	(fugiat)
Mx 	342.05103
T	78 85 76 76 65	(nulla)
Mx 	34.82079
My 	369.18485
T	80 65 82 73 65 84 85 82 14	(pariatur.)
Mx 	70.33591
T	37 88 67 69 80 84 69 85 82	(Excepteur)
Mx 	110.76411
T	83 73 78 84	(sint)
Mx 	127.73431
T	79 67 67 65 69 67 65 84	(occaecat)
Mx 	162.56827
T	67 85 80 73 68 65 84 65 84	(cupidatat)
Mx 	200.60585
T	78 79 78	(non)
Mx 	218.05505
T	80 82 79 73 68 69 78 84 12	(proident,)
Mx 	255.30601
T	83 85 78 84	(sunt)
Mx 	274.60530
T	73 78	(in)
Mx 	285.00128
T	67 85 76 80 65	(culpa)
Mx 	308.03154
T	81 85 73	(qui)
Mx 	322.86599
T	79 70 70 73 67 73 65	(officia)
Mx 	348.92408
T	68 69	(de)
Mx 	357.76148
T	13	(-)
Mx 	34.82079
My 	382.18485
T	83 69 82 85 78 84	(serunt)
Mx 	61.24801
T	77 79 76 76 73 84	(mollit)
Mx 	85.81635
T	65 78 73 77	(anim)
Mx 	106.97893
T	73 68	(id)
Mx 	116.49600
T	69 83 84	(est)
Mx 	129.62538
T	76 65 66 79 82 85 77 14	(laborum.)
Mx 	34.82079
My 	398.67610
T	19 14	(3.)
Mx 	59.10497
T	38 79 79 84 78 79 84 69	(Footnote)
Mx 	95.13531
T	84 72 82 69 69 14	(three.)
Mx 	119.96838
T	44 79 82 69 77	(Lorem)
Mx 	146.55927
T	73 80 83 85 77	(ipsum)
Mx 	172.06911
T	68 79 76 79 82	(dolor)
Mx 	194.63460
T	83 73 84	(sit)
Mx 	206.47745
T	65 77 69 84 12	(amet,)
Mx 	229.98337
T	67 79 78 83 69 67 84 69 84 85 82	(consectetur)
Mx 	276.49028
T	65 68 73 80 73 83 73 67 73 78 71	(adipisicing)
Mx 	319.42003
T	69 76 73 84 12	(elit,)
Mx 	336.44403
T	83 69 68	(sed)
Mx 	351.58716
T	68 79	(do)
Mx 	34.82079
My 	411.67610
T	69 73 85 83 77 79 68	(eiusmod)
Mx 	68.15701
T	84 69 77 80 79 82	(tempor)
Mx 	97.56012
T	73 78 67 73 68 73 68 85 78 84	(incididunt)
Mx 	138.02867
T	85 84	(ut)
Mx 	147.95522
T	76 65 66 79 82 69	(labore)
Mx 	173.40326
T	69 84	(et)
Mx 	182.71897
T	68 79 76 79 82 69	(dolore)
Mx 	208.67238
T	77 65 71 78 65	(magna)
Mx 	235.57940
T	65 76 73 81 85 65 14	(aliqua.)
Mx 	262.24472
T	53 84	(Ut)
Mx 	273.28748
T	69 78 73 77	(enim)
Mx 	294.13884
T	65 68	(ad)
Mx 	305.01022
T	77 73 78 73 77	(minim)
Mx 	331.37232
T	86 69 78 73 65 77 12	(veniam,)
Mx 	34.82079
My 	424.67610
T	81 85 73 83	(quis)
Mx 	52.28427
T	78 79 83 84 82 85 68	(nostrud)
Mx 	83.50703
T	69 88 69 82 67 73 84 65 84 73 79 78	(exercitation)
Mx 	130.68633
T	85 76 76 65 77 67 79	(ullamco)
Mx 	162.29581
T	76 65 66 79 82 73 83	(laboris)
Mx 	189.58546
T	78 73 83 73	(nisi)
Mx 	205.04943
T	85 84	(ut)
Mx 	215.05979
T	65 76 73 81 85 73 80	(aliquip)
Mx 	242.82405
T	69 88	(ex)
Mx 	253.65179
T	69 65	(ea)
Mx 	264.08403
T	67 79 77 77 79 68 79	(commodo)
Mx 	302.87856
T	67 79 78 83 69 81 85 65 84 14	(consequat.)
Mx 	344.77564
T	36 85 73 83	(Duis)
Mx 	34.82079
My 	437.67610
T	65 85 84 69	(aute)
Mx 	54.11716
T	73 82 85 82 69	(irure)
Mx 	75.75142
T	68 79 76 79 82	(dolor)
Mx 	98.62933
T	73 78	(in)
Mx 	109.17619
T	82 69 80 82 69 72 69 78 68 69 82 73 84	(reprehenderit)
Mx 	164.55605
T	73 78	(in)
Mx 	175.10291
T	86 79 76 85 80 84 65 84 69	(voluptate)
Mx 	213.66930
T	86 69 76 73 84	(velit)
Mx 	233.38315
T	69 83 83 69	(esse)
Mx 	251.79182
T	67 73 76 76 85 77	(cillum)
Mx 	278.18096
T	68 79 76 79 82 69	(dolore)
Mx 	305.21610
T	69 85	(eu)
Mx 	317.28347
T	70 85 71 73 65 84	(fugiat)
Mx 	342.05103
T	78 85 76 76 65	(nulla)
Mx 	34.82079
My 	450.67610
T	80 65 82 73 65 84 85 82 14	(pariatur.)
Mx 	70.33591
T	37 88 67 69 80 84 69 85 82	(Excepteur)
Mx 	110.76411
T	83 73 78 84	(sint)
Mx 	127.73431
T	79 67 67 65 69 67 65 84	(occaecat)
Mx 	162.56827
T	67 85 80 73 68 65 84 65 84	(cupidatat)
Mx 	200.60585
T	78 79 78	(non)
Mx 	218.05505
T	80 82 79 73 68 69 78 84 12	(proident,)
Mx 	255.30601
T	83 85 78 84	(sunt)
Mx 	274.60530
T	73 78	(in)
Mx 	285.00128
T	67 85 76 80 65	(culpa)
Mx 	308.03154
T	81 85 73	(qui)
Mx 	322.86599
T	79 70 70 73 67 73 65	(officia)
Mx 	348.92408
T	68 69	(de)
Mx 	357.76148
T	13	(-)
Mx 	34.82079
My 	463.67610
T	83 69 82 85 78 84	(serunt)
Mx 	61.36886
T	77 79 76 76 73 84	(mollit)
Mx 	86.05805
T	65 78 73 77	(anim)
Mx 	107.34148
T	73 68	(id)
Mx 	116.97940
T	69 83 84	(est)
Mx 	130.22963
T	76 65 66 79 82 85 77 14	(laborum.)
Mx 	166.03698
T	44 79 82 69 77	(Lorem)
Mx 	192.31699
T	73 80 83 85 77	(ipsum)
Mx 	217.51595
T	68 79 76 79 82	(dolor)
Mx 	239.77057
T	83 73 84	(sit)
Mx 	251.30253
T	65 77 69 84 12	(amet,)
Mx 	274.49758
T	67 79 78 83 69 67 84 69 84 85 82	(consectetur)
Mx 	320.69361
T	65 68 73 80 73 83 73 67 73 78 71	(adipisicing)
Mx 	34.82079
My 	476.67610
T	69 76 73 84 12	(elit,)
Mx 	52.21464
T	83 69 68	(sed)
Mx 	67.72763
T	68 79	(do)
Mx 	80.13369
T	69 73 85 83 77 79 68	(eiusmod)
Mx 	114.60909
T	84 69 77 80 79 82	(tempor)
Mx 	145.15138
T	73 78 67 73 68 73 68 85 78 84	(incididunt)
Mx 	186.75910
T	85 84	(ut)
Mx 	197.82482
T	76 65 66 79 82 69	(labore)
Mx 	224.41204
T	69 84	(et)
Mx 	234.86692
T	68 79 76 79 82 69	(dolore)
Mx 	261.95951
T	77 65 71 78 65	(magna)
Mx 	290.00570
T	65 76 73 81 85 65 14	(aliqua.)
Mx 	317.81020
T	53 84	(Ut)
Mx 	329.99214
T	69 78 73 77	(enim)
Mx 	351.98267
T	65 68	(ad)
Mx 	34.82079
My 	489.67610
T	77 73 78 73 77	(minim)
Mx 	61.43735
T	86 69 78 73 65 77 12	(veniam,)
Mx 	93.17353
T	81 85 73 83	(quis)
Mx 	110.80767
T	78 79 83 84 82 85 68	(nostrud)
Mx 	142.20108
T	69 88 69 82 67 73 84 65 84 73 79 78	(exercitation)
Mx 	189.55104
T	85 76 76 65 77 67 79	(ullamco)
Mx 	221.33117
T	76 65 66 79 82 73 83	(laboris)
Mx 	248.79148
T	78 73 83 73	(nisi)
Mx 	264.42610
T	85 84	(ut)
Mx 	274.60712
T	65 76 73 81 85 73 80	(aliquip)
Mx 	302.54203
T	69 88	(ex)
Mx 	313.54043
T	69 65	(ea)
Mx 	324.14331
T	67 79 77 77 79 68 79	(commodo)
[1] Mx 	195.46105
My 	519.80194
Set font 	Gentium;10;200;regular;normal;;LTR
T	17	(1)
New page
Mx 	78.73386
My 	443.32904
Set font 	Gentium;10;200;italic;normal;;LTR
T	51 69 80 65 82 65 84 79 82	(Separator)
Mx 	58.73386
My 	466.75520
Set font 	Gentium;9;200;regular;normal;;LTR
T	67 79 78 83 69 81 85 65 84 14	(consequat.)
Mx 	101.35091
T	36 85 73 83	(Duis)
Mx 	120.23309
T	65 85 84 69	(aute)
Mx 	139.25150
T	73 82 85 82 69	(irure)
Mx 	160.60781
T	68 79 76 79 82	(dolor)
Mx 	183.20776
T	73 78	(in)
Mx 	193.47667
T	82 69 80 82 69 72 69 78 68 69 82 73 84	(reprehenderit)
Mx 	248.57857
T	73 78	(in)
Mx 	258.84748
T	86 79 76 85 80 84 65 84 69	(voluptate)
Mx 	297.13591
T	86 69 76 73 84	(velit)
Mx 	316.57180
T	69 83 83 69	(esse)
Mx 	334.70252
T	67 73 76 76 85 77	(cillum)
Mx 	360.81371
T	68 79 76 79 82 69	(dolore)
Mx 	58.73386
My 	479.75520
T	69 85	(eu)
Mx 	70.44087
T	70 85 71 73 65 84	(fugiat)
Mx 	94.84808
T	78 85 76 76 65	(nulla)
Mx 	116.37247
T	80 65 82 73 65 84 85 82 14	(pariatur.)
Mx 	151.67812
T	37 88 67 69 80 84 69 85 82	(Excepteur)
Mx 	191.89685
T	83 73 78 84	(sint)
Mx 	208.65757
T	79 67 67 65 69 67 65 84	(occaecat)
Mx 	243.28207
T	67 85 80 73 68 65 84 65 84	(cupidatat)
Mx 	281.11017
T	78 79 78	(non)
Mx 	298.34990
T	80 82 79 73 68 69 78 84 12	(proident,)
Mx 	335.39138
T	83 85 78 84	(sunt)
Mx 	354.48121
T	73 78	(in)
Mx 	364.66771
T	67 85 76 80 65	(culpa)
Mx 	58.73386
My 	492.75520
T	81 85 73	(qui)
Mx 	72.97024
T	79 70 70 73 67 73 65	(officia)
Mx 	98.43026
T	68 69 83 69 82 85 78 84	(deserunt)
Mx 	133.69008
T	77 79 76 76 73 84	(mollit)
Mx 	158.25361
T	65 78 73 77	(anim)
Mx 	179.41138
T	73 68	(id)
Mx 	188.92365
T	69 83 84	(est)
Mx 	202.04822
T	76 65 66 79 82 85 77 14	(laborum.)
[2] Mx 	219.37413
My 	519.75311
Set font 	Gentium;10;200;regular;normal;;LTR
T	18	(2)
End page
Finish

